core/engine/src/builtins/intl/number_format/options.rs

```diff
@@ -180,3 +1208,48 @@ impl RoundingType {
         }
     }
 }
+
+#[cfg(test)]
```

Comment:
nit: Separate the tests in to a `tests.rs` file.

With a project this large, it feels like it would be worthwhile to have the tests a bit more visible in their own file instead of in the file itself. If a potential contributor came in and wanted to add tests, they might not know that there were some tests already in place at a glance.